app/sagas/cpacpy.default.js

```diff
@@ -0,0 +1,11 @@
+import {fromJS} from "immutable";
+import { scheduler } from 'consts'
+
+export const cpacpyConfig = fromJS({
+  schedulers: [
+    { id: 'local', name: 'Local', version: 'unknown', backends: [], address: scheduler.local, polling: false, detecting: false, online: null, connected: false, connect: { callbacks: {} } },
+  ],
+  // id for scheduler
+  scheduler: 'local',
```

Comment:
To remove

Reply:
My suggestion is to keep it - we need to provide a default loading template for the `configLocalState` method for `sagas/cpacpy` module.

Reply:
LGTM!